scss/_patterns_side-navigation.scss

```diff
@@ -539,7 +539,7 @@
     .p-side-navigation--raw-html {
       @include vf-side-navigation-raw-html-theme-light;
 
-      &.is-light {
+      &.is-dark {
```

Comment:
Drive-by: fixed a broken class name when switching from light theme to dark theme in raw HTML variant.